src/components/ui/AppColorPicker.vue

```diff
@@ -154,6 +164,8 @@ export default class AppColorPicker extends Vue {
   dot!: boolean
 
   menu = false
+  dragging = false
+  lastPointerPosition: {x: number, y: number} = { x: 0, y: 0 }
```

Comment:
As you have a couple of places with `{x: number, y: number}`, probably best to just create a local type and reuse that?